Linear_Search/Linear_Search.kt

```diff
@@ -0,0 +1,39 @@
+//Function For Linear Search
+fun Linear_Search(list: List<Int>, Num: Int):Int{
+    //If number not found than return -1 otherwise return postion of that number.
+    var foundAt:Int = -1
+    for(number in 0 until list.size){
```

Comment:
Add a space between ')' and '{'

Reply:
Done @amaaniqbal

Reply:
Done

Reply:
I still see them at quite a few places. Line 5, 19, 28, 31 etc.

Reply:
Done